plugins/module_utils/_ADObject.psm1

```diff
@@ -543,6 +543,9 @@ Function ConvertTo-AnsibleADDistinguishedName {
         [string]
         $Server,
 
+        [boolean]
+        $NestedGroupFlatten,
```

Comment:
This should probably be a `[Switch]` rather than a boolean.

Reply:
Agreed, done.